ApeWorX/ape
# fix: help text fixes

### What I did

* Corrects help text in yes options for plugins (said “remove” in places where it should have said “add”)
* Adds missing help text for the verbose option in the `ape console` command
* Adds shorthand `-v` to the verbose option in the `ape console` command
* Moves shared CLI options to an `options.py` module, including the `network_option` plus its choice class

### How I did it

Noticed that `NetworkChoice` was being imported from one 1st class plugin to another, and that had a bad smell to it.
Sought to move it to a share module for plugins to use, but then noticed that the `--yes` option in the `plugins` command had mistakenly said `remove` when it should have said `add`, so fixed that, since I was analyzing shared-options anyway, it felt like it fit. I then noticed that the verbose option could also be shared but more importantly, it was missing a `help` text in the `ape console` command.

### How to verify it

1.) Do

```
ape console --help
```

Notice there is now a help text for the `--verbose` options.

2.) Do

```
ape plugins add --help
ape plugins remove --help
ape plugins install --help
```

Notice each of the `--yes` option makes sense per the command (before, they all just spoke like it was for the `remove` command).

### Checklist
